ros/src/computing/perception/detection/lidar_tracker/packages/lidar_imm_ukf_pda_track/include/imm_ukf_pda.h

```diff
@@ -126,6 +126,7 @@ class ImmUkfPda
   void mergeOverSegmentation(const std::vector<UKF> targets);
 
   void updateJskLabel(const UKF& target, jsk_recognition_msgs::BoundingBox& bb);
+  void updateBehaviorState(const UKF& target, autoware_msgs::DetectedObject& dd);
```

Comment:
Variable name is important. The reader will drop off what `bb` and `dd` mean in the function.

Reply:
Please check the function. It is not too long. 
So reader would not forget what dd is.

Reply:
but will modify here

Reply:
I did change for clarification

Reply:
It is not a good idea to change the policy based on function length. Please note that reading code consumes 10, 100, or more times than writing code. The mind making code more readable extends Autoware world :)